Tests/CoreTests/ISO8601Tests.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import XCTest
+@testable import Core
+
+class ISO8601Tests: XCTestCase {
+
+    static var allTests = [
+        ("testBasic", testBasic)
+    ]
```

Comment:
I thought so too, but the test seemed to run and I copied this from https://github.com/vapor/core/blob/master/Tests/CoreTests/RFC1123Tests.swift#L8, should I add it to that test as well?